### PR TITLE
Wip/work on misc functionality

### PR DESCRIPTION
**Pull request by bendavidsonku**

### CODE REVIEW

File: YourToDo/templates/planner/ajax_events_in_planner_month_view.html [hunk 1]
```diff
@@ -72,7 +72,8 @@
     </tr>
     <tr class="planner-month-sidebar-info">
         <td>
-            Events that you have recently marked as 'completed' will show up here.
+            <div id="recently-completed-list">
```
Comment (JayOfferdahl) [inline YourToDo/templates/planner/ajax_events_in_planner_month_view.html:75]: :+1:

Comment (bendavidsonku) [inline YourToDo/templates/planner/ajax_events_in_planner_month_view.html:75]: thx

File: YourToDo/templates/planner/ajax_important_and_upcoming.html [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
     	{% if importantEvent.get_complete != True %}
-    		<div class="important-and-upcoming-event">
+        <div>
+    		<div class="important-and-upcoming-event update-event-modal-button" id="{{ importantEvent.get_event_id }}">
     			<span class="planner-category-color-{{ importantEvent.get_parentCategory.get_color }} important-and-upcoming-event-span">
-    		    	{{ importantEvent.get_parentCategory.get_name }}: {{ importantEvent.get_name }} - {{ importantEvent.get_dateOfEvent }}
+    		    	{{ importantEvent.get_parentCategory.get_name }}: {{ importantEvent.get_name }}
     		    </span>
     	    </div>
+            &nbsp;- {{ importantEvent.get_dateOfEvent }}
```
Comment (JayOfferdahl) [inline YourToDo/templates/planner/ajax_important_and_upcoming.html:10]: As per offline discussion, I'll make a method to tear the ", 20XX" off the back of these dates.

**Comment by JayOfferdahl** [comment]
Looks good!

**Pull request merged**